render/render.go

```diff
@@ -178,7 +178,7 @@ func fitText(text string, fontSize int, W float64) (scale float64, h float64) {
 }
 
 func textWidth(text string, fontSize float64) float64 {
-	return fontSize * float64(len(text)) * textWidthMultiplier
+	return fontSize * float64(len([]rune(text))) * textWidthMultiplier
```

Comment:
nice

could you use this instead? https://pkg.go.dev/unicode/utf8#RuneCountInString

Reply:
Cool, thanks! Was not aware of that, also already asking myself if it isn't a waste of resources that string length needs to be calculates like this in Go. Will do.

Reply:
Done.

Reply:
that's UTF8, so it is O(N). good to have some optimisation! tbh I see lots of non-obviously complex things are done really well by Go core (other example is integer parsing/printing). anws lgtm!